torch/_refs/__init__.py

```diff
@@ -852,7 +879,7 @@ def float_power(
         b = prims.to_dtype(b, dtype)
 
     a, b = _maybe_broadcast(a, b)
-    return prims.pow(a, b)
+    return pow(a, b)
```

Comment:
`a**b`

Reply:
This was failing with some error. Leaving it as `pow(a, b)`.

Reply:
fwiw it throws a `RuntimeError: no _refs support for torch.Tensor.__pow__`